ui/imports/shared/controls/chat/LinkPreviewCard.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared.status 1.0
+
+CalloutCard {
+    id: root
+    
+    /// The title of the callout card
+    required property string title
+    required property string description
+    required property string footer
```

Comment:
Also, maybe we could got with inheritance solution and create a separate components derived from `LinkPreviewCard`:
- `CommonLinkPreviewCard`
- `StatusCommunityPreviewCard`
- `StatusChannelPreviewCard`
- `ImagePreviewCard`

Just an idea, it might be not quite convenient to use.

Reply:
> Also, maybe we could got with inheritance solution and create a separate components derived from LinkPreviewCard:

I've explored a bit this idea, but found it quite inconvenient. We'd need a loader to load the proper component based on some conditions. I'd like to avoid that.

> Are you sure that passing html here is a good idea? It seems better to me to have properties smth like this:

It made sense to me because the footer is quite simple: one row of text and occasionally some inline icons. RichText can easely handle this.
I totally agree the footer content should be built inside the card component based on some specific properties. I used to think about it as a generic component that doesn't need to know about communities and links.

Reply:
> I totally agree the footer content should be built inside the card component based on some specific properties. I used to think about it as a generic component that doesn't need to know about communities and links.

@igor-sirotin I'll do it in a next PR when integrating the community cards if that's ok with you. It will probably need an update when the backend for the community cards is ready so we might as well polish the properties when the model is ready. The behaviour is implemented, it will be just a matter of finding the best solution based on the backend design.

Reply:
https://github.com/status-im/status-desktop/issues/12176 I've added a task here